@app/server/src/middleware/installSSR.ts

```diff
@@ -1,17 +1,9 @@
-// TODO: fix to 'import next' when next fixes the bug
+import { isDev, ROOT_URL, T_AND_C_URL } from "@app/config";
 import { Express } from "express";
 import next from "next";
 import { parse } from "url";
 
-if (!process.env.NODE_ENV) {
-  throw new Error("No NODE_ENV envvar! Try `export NODE_ENV=development`");
-}
-
-const isDev = process.env.NODE_ENV === "development";
-
 export default async function installSSR(app: Express) {
-  // @ts-ignore Next had a bad typing file, they claim `export default` but should have `export =`
-  // Ref: https://unpkg.com/next@9.0.3/dist/server/next.js
```

Comment:
This and the comment at the top of this file seem to have been resolved. Also `// Foo` below may have been resolved 🤔 .